docker-for-mac/faqs.md

```diff
@@ -154,6 +154,13 @@ response.
 If you do not want auto-send of usage data, use the Stable channel. For more
 information, see [Stable and Edge channels](#stable-and-edge-channels) ("What is the difference between the Stable and Edge versions of Docker for Mac?").
 
+### How is personal data handled in Docker Desktop?
+
+When uploading diagnostics to help Docker with investigating issues, the uploaded diagnostics bundle may contain personal data such as usernames and IP addresses.
+The diagnostics bundles are only accessible to a Docker Inc. employees who are directly involved in diagnosing Docker Desktop issues.
```

Comment:
remove the "a" before Docker Inc.

Reply:
done

Reply:
@gtardiff did you push that change? I still see it :sweat_smile: